Replace WebSecurityConfigurerAdapter configuration with SecurityFilterChain bean (#3499)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/configcors.md

> 1. To configure CORS in Spring Security, enable it in the `SecurityFilterChain` that you defined in the previous step:

Reviewer: ```suggestion
1. To configure CORS in Spring Security, enable it in a `SecurityFilterChain` bean:
```
I don't think there is a previous step anymore, is there?

Author: I think this is still one, it refers to the previous section in the docs (Configure required authentication) where the WebSecurityConfigurerAdapter is defined for the first time. The first step becomes now the place where the SecurityFilterChain is defined, so unless I am wrong this reference is still valid.
Have a look here 
  https://developer.okta.com/docs/guides/protect-your-api/springboot/main/#configure-required-authentication

   ```java
    @Bean

>     SecurityFilterChain oauth2SecurityFilterChain2(HttpSecurity http) throws Exception {

Reviewer: ```suggestion
    SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Simplify name as shown in https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter.
